### DISCUSSION TITLE
Mechanical Optical Switches

### DISCUSSION THREAD

**Discussion by MaWagener** [iontraps/awesome-iontraps#9] [Q&A]
I was wondering if we could use fast mechanical switches instead of AOMs for certain applications. For example I would like a to switch a 397nm pre-cooling beam during an experimental sequence. So switching on and off the zeroth order of a double pass would be nice for that. As far as understand people don't usually do that because it is slow compared to an AOM. However, it might be possible to get to well below a ms using a small mirror on a piezo I think. 
Any thoughts on that? Am I missing something?

**Comment by chizhang1990** [comment]
Hi, I had a similar thought before and also searched a bit about the mechanical shutters. However, I could not find any mechanical shutter that is comparably fast as an AOM. Millisecond switching time is still too slow for use in the experiment sequence. 

Besides, I found it not so easy to place the commercial fast shutters (like those: https://www.uniblitz.com/shutter-series-overview/) into the beam path without blocking the 1st order beam. In the closed state, they typically fully block the lasers beams, including the 1st-order beam. This problem can be solved, in principle, by customizing the shutter. But not really sure this would be easier than using an AOM.

**Reply by MaWagener** [reply]
I was thinking about something like this rather than a classical shutter.
https://www.piezosystem.com/products/piezo_actuators/mirror_tilting_systems/psh_41/

This way you can just introduce a slight tilt on the laser which should be enough to miss-align the fiber coupling. Also, this does not need to be in the double-pass beam path directly. You could block the 0th order with a D-shaped mirror and than retro reflect with this piezo thing. Or maybe block the 0th order with a mirror on a piezo directly.

You can get piezos with 10s of KHz resonant frequency (not sure how much with a mirror). Not quite sure what that means for switching times directly, but I would think sub-ms.

**Reply by chizhang1990** [reply]
This looks interesting. It could also be used to steer the final beam pointing before the vacuum chamber (for other applications).
Would be good to know its price and robustness.
